third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt

```diff
@@ -0,0 +1,12 @@
+Tests that exporting works.
```

Comment:
this is the diff between ToT chromium and these rebaselined tests:

```sh
git diff --no-index ~/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/ ./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/
```

```diff
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/OWNERS b/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/OWNERS
deleted file mode 100644
index adbdad6cd..000000000
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/OWNERS
+++ /dev/null
@@ -1 +0,0 @@
-file://LIGHTHOUSE_OWNERS
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt
index 596f107d8..30b8393bf 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt
@@ -2,11 +2,11 @@ Tests that exporting works.
 
 ++++++++ testExportHtml
 
-# of .lh-audit divs (original): 133
+# of .lh-audit divs (original): 135
 
-# of .lh-audit divs (exported html): 133
+# of .lh-audit divs (exported html): 135
 
 ++++++++ testExportJson
 
-# of audits (json): 149
+# of audits (json): 151
 
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-flags-run-expected.txt b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-flags-run-expected.txt
index 04281a8bb..3eaaed94d 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-flags-run-expected.txt
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-flags-run-expected.txt
@@ -18,6 +18,3 @@ disableStorageReset: false
 throttlingMethod: devtools
 TestedAsMobileDevice: false
 
-View Trace Button Text: "View Trace"
-View Trace Button Title: ""
-
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-flags-run.js b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-flags-run.js
index 2ade417e5..4a8282f95 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-flags-run.js
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-flags-run.js
@@ -26,9 +26,5 @@
   TestRunner.addResult(`throttlingMethod: ${lhr.configSettings.throttlingMethod}`);
   TestRunner.addResult(`TestedAsMobileDevice: ${artifacts.TestedAsMobileDevice}`);
 
-  const viewTraceButton = LighthouseTestRunner.getResultsElement().querySelector('.view-trace');
-  TestRunner.addResult(`\nView Trace Button Text: "${viewTraceButton.textContent}"`);
-  TestRunner.addResult(`View Trace Button Title: "${viewTraceButton.title}"`);
-
   TestRunner.completeTest();
 })();
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt
index c7946b65c..320e12240 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt
@@ -243,6 +243,8 @@ Auditing: Avoid long main-thread tasks
 Auditing: Avoids `unload` event listeners
 Auditing: Avoid non-composited animations
 Auditing: Image elements have explicit `width` and `height`
+Auditing: Avoids large JavaScript libraries with smaller alternatives
+Auditing: Page has valid source maps
 Auditing: Site works cross-browser
 Auditing: Page transitions don't feel like they block on the network
 Auditing: Each page has a URL
@@ -425,6 +427,7 @@ is-crawlable: pass
 is-on-https: pass
 js-libraries: pass
 label: notApplicable
+large-javascript-libraries: pass
 largest-contentful-paint: numeric
 largest-contentful-paint-element: informative
 layout-shift-elements: notApplicable
@@ -501,6 +504,7 @@ uses-responsive-images: flaky
 uses-text-compression: flaky
 uses-webp-images: flaky
 valid-lang: notApplicable
+valid-source-maps: pass
 video-caption: notApplicable
 video-description: notApplicable
 viewport: fail
@@ -508,5 +512,5 @@ visual-order-follows-dom: manual
 without-javascript: pass
 works-offline: fail
 
-# of .lh-audit divs: 133
+# of .lh-audit divs: 135
 
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt
index 7340ef036..81647fd29 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt
@@ -27,6 +27,7 @@ first-cpu-idle
 first-meaningful-paint
 font-display
 interactive
+large-javascript-libraries
 largest-contentful-paint
 largest-contentful-paint-element
 layout-shift-elements
@@ -69,8 +70,5 @@ uses-responsive-images
 uses-text-compression
 uses-webp-images
 
-View Trace Button Text: "View Original Trace"
-View Trace Button Title: "The performance metrics above are simulated and won't match the timings found in this trace. Disable simulated throttling in "Lighthouse Settings" if you want the timings to match."
-
 Showing view: timeline
 
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run.js b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run.js
index 2f5e3c314..4b8e6a721 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run.js
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run.js
@@ -32,8 +32,6 @@
     TestRunner.addSniffer(UI.ViewManager.prototype, 'showView', resolve);
   });
   const viewTraceButton = LighthouseTestRunner.getResultsElement().querySelector('.view-trace');
-  TestRunner.addResult(`\nView Trace Button Text: "${viewTraceButton.textContent}"`);
-  TestRunner.addResult(`View Trace Button Title: "${viewTraceButton.title}"`);
   viewTraceButton.click();
   const viewShown = await waitForShowView;
   TestRunner.addResult(`\nShowing view: ${viewShown}`);
```

Reply:
ah... actually looks like you need to roll the tests from devtools repo to LH again.. the traceButton change strikes.

Reply:
ah yea, @adamraine recently added that

Reply:
new commit who diff

```diff
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/OWNERS b/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/OWNERS
deleted file mode 100644
index adbdad6cd..000000000
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/OWNERS
+++ /dev/null
@@ -1 +0,0 @@
-file://LIGHTHOUSE_OWNERS
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt
index 596f107d8..57304d5d2 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt
@@ -2,11 +2,11 @@ Tests that exporting works.
 
 ++++++++ testExportHtml
 
-# of .lh-audit divs (original): 133
+# of .lh-audit divs (original): 136
 
-# of .lh-audit divs (exported html): 133
+# of .lh-audit divs (exported html): 136
 
 ++++++++ testExportJson
 
-# of audits (json): 149
+# of audits (json): 152
 
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt
index c7946b65c..5d886e2e0 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt
@@ -216,6 +216,7 @@ Auditing: Manifest has a maskable icon
 Auditing: Content is sized correctly for the viewport
 Auditing: Displays images with correct aspect ratio
 Auditing: Serves images with appropriate resolution
+Auditing: Fonts with `font-display: optional` are preloaded
 Auditing: Avoids deprecated APIs
 Auditing: Minimizes main-thread work
 Computing artifact: MainThreadTasks
@@ -243,6 +244,8 @@ Auditing: Avoid long main-thread tasks
 Auditing: Avoids `unload` event listeners
 Auditing: Avoid non-composited animations
 Auditing: Image elements have explicit `width` and `height`
+Auditing: Avoids large JavaScript libraries with smaller alternatives
+Auditing: Page has valid source maps
 Auditing: Site works cross-browser
 Auditing: Page transitions don't feel like they block on the network
 Auditing: Each page has a URL
@@ -425,6 +428,7 @@ is-crawlable: pass
 is-on-https: pass
 js-libraries: pass
 label: notApplicable
+large-javascript-libraries: pass
 largest-contentful-paint: numeric
 largest-contentful-paint-element: informative
 layout-shift-elements: notApplicable
@@ -461,6 +465,7 @@ offscreen-images: flaky
 password-inputs-can-be-pasted-into: pass
 performance-budget: notApplicable
 plugins: pass
+preload-fonts: notApplicable
 pwa-cross-browser: manual
 pwa-each-page-has-url: manual
 pwa-page-transitions: manual
@@ -501,6 +506,7 @@ uses-responsive-images: flaky
 uses-text-compression: flaky
 uses-webp-images: flaky
 valid-lang: notApplicable
+valid-source-maps: pass
 video-caption: notApplicable
 video-description: notApplicable
 viewport: fail
@@ -508,5 +514,5 @@ visual-order-follows-dom: manual
 without-javascript: pass
 works-offline: fail
 
-# of .lh-audit divs: 133
+# of .lh-audit divs: 136
 
diff --git a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt
index 7340ef036..8628b1c89 100644
--- a/Users/paulirish/chromium-devtools/devtools-frontend/test/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt
+++ b/./third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt
@@ -27,6 +27,7 @@ first-cpu-idle
 first-meaningful-paint
 font-display
 interactive
+large-javascript-libraries
 largest-contentful-paint
 largest-contentful-paint-element
 layout-shift-elements

```

LGTM